app/models/drawing.rb

```diff
@@ -4,6 +4,7 @@ class Drawing < ActiveRecord::Base
 
   validates :image, presence: true
   validates :status, presence: true
+  validates_inclusion_of :image_consent, in: [true, false]
```

Comment:
This actually isn't required as front end forces this to be true or false, but it a failsafe to ensure one is picked.